src/preprocessing.py

```diff
@@ -33,18 +34,24 @@ def preprocessing(df):
         'security_test_result (FALSE=Passed; TRUE=Failed)': 'security_test_result',
         'risk_label_Baljeet': 'risk_label'
         }, inplace=True)
+    # Merge country_metric data with input data
+    df = df.merge(country_metric_df, left_on='server_location', right_on='Country')
     # Drop the columns that are not needed
-    df = df.drop(['response_metadata'], axis=1) # To-be-modifited
+    df = df.drop(['response_metadata', 'server_location'], axis=1) # To-be-modifited
     # Update the rows with duplicates
     df.loc[df['hosting_isp'] == 'Amazon Technologies Inc.', 'hosting_isp'] = 'Amazon.com, Inc.'
     # Drop the rows with duplicates
     df = df.drop_duplicates()
     return df
 
-def main(input_path, output_path):
+def main(input_path, input_path_country, output_path):
     # Read the file
     df = pd.read_excel(input_path, "Core_Endpoint", usecols = "A:R")
 
+    # Read country metric data
+    country_metric_df = pd.read_excel(input_path_country, "NRI 2021 - results", usecol = "B:C", skiprows=1) 
+    # To-be-modified once we decide which metrics to use, presently, just the overall score
+
     # Preprocess the data
     df = preprocessing(df)
```

Comment:
Added the input argument
```suggestion
    df = preprocessing(df, country_metric_df)
```